spec/asyncapi.md

```diff
@@ -918,6 +929,17 @@ bindings:
 
 
 
+#### <a name="operationReplyObject"></a>Reply Object
```

Comment:
Yes in mean while i would agree.  Will be changed

Reply:
Please see my comment before you change it https://github.com/asyncapi/spec/pull/847#discussion_r992500602

Reply:
On hold as long we have no consensus at: 
https://github.com/asyncapi/spec/pull/847#discussion_r1003458287

because if we get `operation[x].messages´ it would also make sense to have it below `reply`